src/commands/createNewProject/pythonSteps/pythonVersion.ts

```diff
@@ -23,22 +24,43 @@ export async function getPythonVersion(pyAlias: string): Promise<string> {
     }
 }
 
-export async function getSupportedPythonVersions(): Promise<string[]> {
-    const versions: string[] = ['3.6'];
+export async function getSupportedPythonVersions(funcVersionFromSetting: FuncVersion): Promise<string[]> {
+    // Cache the task so we're not waiting on this every time
+    const localVersionTask: Promise<string | null> = getLocalFuncCoreToolsVersion();
 
-    const minFuncVersion: string = '2.7.1846';
-    try {
-        const currentVersion: string | null = await getLocalFuncCoreToolsVersion();
-        if (currentVersion && semver.lt(currentVersion, minFuncVersion)) {
-            ext.outputChannel.appendLine(localize('outOfDateWarning', 'WARNING: Some Python versions are not supported with version "{0}" of the func CLI. Update to at least "{1}".', currentVersion, minFuncVersion));
-            return versions;
+    const result: string[] = [];
+
+    const versionInfo: [string, string][] = [
+        ['2.0.0', '3.6'],
```

Comment:
Why is this a 2d array now?

Reply:
The first entry is minimum func cli version and the second entry is the python version it supports. It's more clear a few lines down in the for loop. The reason I need a 2d array now is just because it gets more and more complicated when I have to add more python versions